src/js/hooks/useFocusOnMatch.js

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react';
+
+import cycleCountColumn from 'consts/cycleCountColumn';
+
+const useFocusOnMatch = ({
```

Comment:
Is this one CycleCount specific or should be generic? If `CC` specific, then move to a proper folder, otherwise please make it generic.